src/smpp_gateway/router.py

```diff
@@ -0,0 +1,35 @@
+from rapidsms.messages.incoming import IncomingMessage
+from rapidsms.messages.outgoing import OutgoingMessage
+from rapidsms.router.blocking import BlockingRouter
+
+from smpp_gateway.models import MTMessage
+
+
+class PriorityIncomingMessage(IncomingMessage):
+    def respond(self, text, **kwargs):
+        fields = kwargs.get("fields", {})
+        if "priority_flag" not in fields:
+            fields["priority_flag"] = MTMessage.PriorityFlag.LEVEL_2.value
```

Comment:
To make this easier to customize, perhaps we could add a class-level attribute

```
    default_priority = MTMessage.PriorityFlag.LEVEL_2.value
```

And then
```suggestion
            fields["priority_flag"] = self.default_priority
```